docs/index.html

```diff
@@ -64,8 +64,12 @@ <h1 class="sidebar-header">Lifespan filter<span class="sidebar-close"><i class="
               <input type="number" id="to">
             </div>
             <div class="lifespan-controls">
-              <button id="lifespan-pause"><i class="fas fa-pause"></i></button>
-              <button id="lifespan-play"><i class="fas fa-play"></i></button>
+              <p>To play a timelapse, select the starting year using the "From" handle, then specify window size by adjusting the "To" handle. 
+                The specified window will move forward in with the velocity specified below.</p>
+              <button id="lifespan-backward"><i class="fas fa-backward"></i></button>
+              <button id="lifespan-play"><i id="play-icon" class="fas fa-play"></i></button>
```

Comment:
After reaching 2021, the play does not reappear automatically. Maybe we should write a condition to reset the button, once it reaches the end.

Reply:
Thanks! Fixed (it now legitimately stops when it reaches 2021)